### PR TITLE
[url_launcher] Update readme for iOS targeting iOS 9.0 and higher 

### PR DESCRIPTION
**Pull request by TahaTesser**
## Description

Add instructions for iOS, for targeting iOS 9.0 and higher. 
Note:
All new projects 1.22+ targets iOS 9.0 by default 
Screenshot 2020-11-06 at 4 50 28 PM
97959488-6d67bd00-1dea-11eb-9cdf-a06ea55738d5

## Related Issues

65936

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by pedromassangocode** [comment]
I suggest you to add a explanation why this configuration is required, may be a short explanation and a link to the Apple's new police?

**Comment by TahaTesser** [comment]
> I suggest you to add a explanation why this configuration is required, may be a short explanation and a link to the Apple's new police?

Added explantion

File: packages/url_launcher/url_launcher/README.md [hunk 1]
```diff
@@ -8,6 +8,23 @@ iOS, Android, web, Windows, macOS, and Linux.
...
+
+```
+<key>LSApplicationQueriesSchemes</key>
+<array>
+  <string>https</string>
+  <string>http</string>
+</array>
+```
+
+More details can be found here [canOpenURL](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl).
```
Comment (pedromassangocode) [inline packages/url_launcher/url_launcher/README.md]: I would suggest "See canOpenURL for more details."

Comment (TahaTesser) [reply packages/url_launcher/url_launcher/README.md]: done

**Comment by darshankawar** [comment]
Does it make sense to add a practical example in the description as to where this could be applicable ? Like mentioned in original issue, the device has chrome as default browser. If not set this property in info.plist, the url may not open.

**Comment by TahaTesser** [comment]
> Does it make sense to add a practical example in the description as to where this could be applicable? Like mentioned in the original issue, the device has chrome as the default browser. If not set this property in info.plist, the URL may not open.

It won't break any functionality like opening the link in safari, this was mainly due to targeting iOS 9.0, which is the new default from 1.22+ and 122+ supports also iOS  14 and iOS 14 has default browser settings hence not adding this broke that functionality.  If the team member thinks it would break something, I will make changes. This should be added by default since 1.22+ has been stable for a while

**Comment by darshankawar** [comment]
LGTM

**Review by pedromassangocode — APPROVED** [review]
LGTM

**Pull request merged**